### PR TITLE
Adding The Event Helper Perk

### PR DESCRIPTION
**Pull request by kunalbotla**
this PR adds The Event Helper perk for event insurance in the US. the perk makes the process of obtaining event insurance easier. it links users to an Airtable form that @hackclub/bank-ops creates a quote for using The Hack Foundation's partner link to send back to the submitter for review and purchase with the same method as PVSA.

### CODE REVIEW

**Comment by hackliv** [comment]
1. Thinking we should make the card header the plain orange. 
2. My feedback on rewording: "The Event Helper is partnering with Hack Club Bank to create a simplified way for your team to purchase event insurance. Just apply with your event details to receive a quote from the Bank team. Costs typically start around $60 and increase based on your event's length and number of attendees." &
"The Event Helper can only provide policies for :us: U.S. based events"
3. Looks like the link isn't working when I press purchase

**Comment by hackliv** [comment]
Minor feedback:
"The Event Helper can only provide policies for 🇺🇸 U.S. events." change to "The Event Helper can only provide policies for 🇺🇸 US events." to match PVSA formatting

**Comment by hackliv** [comment]
Hey, did we still want to change the card design per Zach feedback?

**Comment by kunalbotla** [comment]
> Hey, did we still want to change the card design per Zach feedback?

yeah, most of them were on the form. I've updated that there. I'm adding a table to see what it looks like here

**Review by hackliv — APPROVED** [review]
YAY!!!

File: app/assets/stylesheets/components/_promos.scss [hunk 1]
```diff
@@ -57,6 +57,15 @@
   background-repeat: no-repeat;
 }
 
+
+.promo-event-helper {
+  // background-color: $yellow;
+  // background-color: rgb(39, 110, 88);
+  // background-color: rgb(236, 189, 79);
```
Comment (maxwofford) [inline app/assets/stylesheets/components/_promos.scss]:
```suggestion
```

File: app/assets/stylesheets/components/_promos.scss [hunk 2]
```diff
@@ -57,6 +57,15 @@
   background-repeat: no-repeat;
 }
 
+
+.promo-event-helper {
+  // background-color: $yellow;
+  // background-color: rgb(39, 110, 88);
```
Comment (maxwofford) [inline app/assets/stylesheets/components/_promos.scss]:
```suggestion
```

File: app/assets/stylesheets/components/_promos.scss [hunk 3]
```diff
@@ -57,6 +57,15 @@
   background-repeat: no-repeat;
 }
 
+
+.promo-event-helper {
+  // background-color: $yellow;
```
Comment (maxwofford) [inline app/assets/stylesheets/components/_promos.scss]:
```suggestion
```

File: app/views/events/promotions.html.erb [hunk 4]
```diff
@@ -23,6 +23,52 @@
...
+    <%= image_tag 'https://cloud-fz66hmje0-hack-club-bot.vercel.app/0theeventhelperbutbetter.png', width: 60 %>
+    <%= link_to 'Purchase',
+      organizer_signed_in? ? "https://hack.af/event-helper-perk?prefill_Event%20Name=#{@event.name}&prefill_Bank%20URL=#{event_url(@event)}&prefill_Submitter%20Name=#{current_user.full_name}&prefill_Submitter%20Email=#{current_user.email}" : root_path,
+      class: "btn mr1 #{'disabled' unless organizer_signed_in?}",
+      target: :_blank
+    %>
+  </div>
+
+  <p>
+    <%= link_to 'The Event Helper', 'https://www.theeventhelper.com/' %> is partnering with Hack Club Bank to create a simplified way for your team to
```
Comment (maxwofford) [inline app/views/events/promotions.html.erb]:
```suggestion
    <%= link_to 'The Event Helper', 'https://www.theeventhelper.com/', target: :_blank %>
    is partnering with Hack Club Bank to create a simplified way for your team to 
```

File: app/views/events/promotions.html.erb [hunk 5]
```diff
@@ -23,6 +23,52 @@
...
+    <tr>
+      <td>500 people</td>
+      <td>around $200</td>
+    </tr>
+    <tr>
+      <td>1,000 people</td>
+      <td>around $250</td>
+    </tr>
+  </table>
+
```
Comment (maxwofford) [inline app/views/events/promotions.html.erb]:
```suggestion
```

**Comment by exu3** [comment]
It looks like the table hasn't been added to hackclub/airbridge yet:

Screen Shot 2022-06-29 at 11 20 21 AM

**Review by maxwofford — APPROVED** [review]
This looks great! Super clear wording and nice design. I'd like to chat about the content of the form before this is merged, but I'm approving the code changes in this PR.

**Comment by kunalbotla** [comment]
thanks for pointing that out, @exu3! [#68](https://github.com/hackclub/airbridge/pull/68) fixes that!

**Pull request merged**